src/main/java/ru/marina/tshop/orders/orderStatuses/OrderStatus.java

```diff
@@ -1,25 +1,25 @@
-package ru.marina.tshop.orders;
+package ru.marina.tshop.orders.orderStatuses;
```

Comment:
Package names have to be always in lower case letters. See https://www.oracle.com/technetwork/java/codeconventions-135099.html for more details.

> The prefix of a unique package name is always written in all-lowercase ASCII letters